axelor-business-project/src/main/java/com/axelor/apps/businessproject/web/ProjectController.java

```diff
@@ -65,6 +67,22 @@ public void generateQuotation(ActionRequest request, ActionResponse response) {
               .add("form", "sale-order-form")
               .context("_showRecord", String.valueOf(order.getId()))
               .map());
+
+      Partner clientPartner = order.getClientPartner();
+      Boolean interco =
+          order.getCreatedByInterco()
+              || clientPartner == null
+              || Beans.get(CompanyRepository.class)
+                      .all()
+                      .filter("self.partner = ?", clientPartner)
+                      .fetchOne()
+                  == null;
+      Boolean contactPartner =
+          clientPartner != null
+              && clientPartner.getPartnerTypeSelect() == PartnerRepository.PARTNER_TYPE_INDIVIDUAL;
+      response.setHidden("contactPartner", contactPartner);
+      response.setHidden("interco", interco);
+
```

Comment:
Nothing to put for hidden , remove this code , as all this is already done on onLoad. So no onLoad code should be there.